server/src/test/java/org/elasticsearch/index/mapper/DynamicMappingTests.java

```diff
@@ -436,106 +436,6 @@ public void testComplexArray() throws Exception {
                 .endObject().endObject().endObject()), serialize(update));
     }
 
-    public void testReuseExistingMappings() throws IOException, Exception {
```

Comment:
It would be good to match what we have on master (it's easier to reason about, and makes backport conflicts less likely). On master we've modified the test to focus on a single type -- we could apply the same change here.